src/alr/alr-commands.adb

```diff
@@ -461,6 +462,7 @@ package body Alr.Commands is
       if Alire.Platforms.Current.Operating_System /= Alire.Platforms.Windows
         and then not No_Color
         and then not No_TTY
+        and then Ada.Environment_Variables.Value ("TERM", "dumb") /= "dumb"
```

Comment:
Is `dumb` value a usual value for this variable?

Reply:
I'm not sure about the extent of its use but it do appears in `man 7 term` and the emacs shell seems to set it. Whatever IO lib is `cargo` using also reacts to it being set.

Reply:
More than enough for me :)